_scripts/migrations/0026_last_post_not_null.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE discussions
```

Comment:
this should be removed on rebase. tagging to confirm

Reply:
This is fully up to date with master. not sure why we're seeing this collision.

Reply:
Weird -- master doesn't have this file??